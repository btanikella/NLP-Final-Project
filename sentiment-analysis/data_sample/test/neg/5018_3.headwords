wish
movie
But
funny
know
