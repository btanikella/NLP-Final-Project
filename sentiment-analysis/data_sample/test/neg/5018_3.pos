RB DT JJ NN RB VBP PRP VBD VBN PRP IN DT NN
PRP$ DT NN WDT VBZ TO VB JJ IN JJ PRP IN NN TO VB IN  WDT VBZ IN JJS NNS -LRB- NNS IN JJ NNS -RRB-
CC IN DT EX VBZ RB JJ TO DT NN
PRP$ RB JJ CC VBZ PRP JJ RB VB NN NN WDT NN  CC IN FW RB VBD IN NN VBZ RB JJ RBS NNS CC DT CD NN NN  FW VBD RB RB VBN
CC JJS NNS VBP RB VB DT CC FW VBN IN WRB DT NN VBD IN DT NN VBD RP VBD RB JJ IN PRP MD VB VBN CD IN DT JJ NN NN NNS WDT MD VB DT JJ IN DT NN WDT VBZ RB RB DT NN IN -LRB- UH FW VBD PRP$ NN -RRB- NNS IN NNS IN NN IN PRP IN VBG JJ NNS IN PRP MD VB IN WRB PRP MD VB PRP JJR IN VBG PRP IN JJ  VBP NN VBG IN
