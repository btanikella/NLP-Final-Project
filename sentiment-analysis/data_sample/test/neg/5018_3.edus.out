Sentence #1 (14 tokens):
Not a bad concept.I just wish they had taken it in another direction.
[Text=Not CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=RB] [Text=a CharacterOffsetBegin=8 CharacterOffsetEnd=9 PartOfSpeech=DT] [Text=bad CharacterOffsetBegin=10 CharacterOffsetEnd=13 PartOfSpeech=JJ] [Text=concept.I CharacterOffsetBegin=14 CharacterOffsetEnd=23 PartOfSpeech=NN] [Text=just CharacterOffsetBegin=24 CharacterOffsetEnd=28 PartOfSpeech=RB] [Text=wish CharacterOffsetBegin=29 CharacterOffsetEnd=33 PartOfSpeech=VBP] [Text=they CharacterOffsetBegin=34 CharacterOffsetEnd=38 PartOfSpeech=PRP] [Text=had CharacterOffsetBegin=39 CharacterOffsetEnd=42 PartOfSpeech=VBD] [Text=taken CharacterOffsetBegin=43 CharacterOffsetEnd=48 PartOfSpeech=VBN] [Text=it CharacterOffsetBegin=49 CharacterOffsetEnd=51 PartOfSpeech=PRP] [Text=in CharacterOffsetBegin=52 CharacterOffsetEnd=54 PartOfSpeech=IN] [Text=another CharacterOffsetBegin=55 CharacterOffsetEnd=62 PartOfSpeech=DT] [Text=direction CharacterOffsetBegin=63 CharacterOffsetEnd=72 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=72 CharacterOffsetEnd=73 PartOfSpeech=.] 
root(ROOT-0, wish-6)
neg(concept.I-4, Not-1)
det(concept.I-4, a-2)
amod(concept.I-4, bad-3)
nsubj(wish-6, concept.I-4)
advmod(wish-6, just-5)
nsubj(taken-9, they-7)
aux(taken-9, had-8)
ccomp(wish-6, taken-9)
dobj(taken-9, it-10)
prep(taken-9, in-11)
det(direction-13, another-12)
pobj(in-11, direction-13)
punct(wish-6, .-14)

Sentence #2 (29 tokens):
Its a movie that tries to be interesting by shocking you with bestiality to begin with, which works for most people(prudes with weak stomachs).
[Text=Its CharacterOffsetBegin=78 CharacterOffsetEnd=81 PartOfSpeech=PRP$] [Text=a CharacterOffsetBegin=82 CharacterOffsetEnd=83 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=84 CharacterOffsetEnd=89 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=90 CharacterOffsetEnd=94 PartOfSpeech=WDT] [Text=tries CharacterOffsetBegin=95 CharacterOffsetEnd=100 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=101 CharacterOffsetEnd=103 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=104 CharacterOffsetEnd=106 PartOfSpeech=VB] [Text=interesting CharacterOffsetBegin=107 CharacterOffsetEnd=118 PartOfSpeech=JJ] [Text=by CharacterOffsetBegin=119 CharacterOffsetEnd=121 PartOfSpeech=IN] [Text=shocking CharacterOffsetBegin=122 CharacterOffsetEnd=130 PartOfSpeech=JJ] [Text=you CharacterOffsetBegin=131 CharacterOffsetEnd=134 PartOfSpeech=PRP] [Text=with CharacterOffsetBegin=135 CharacterOffsetEnd=139 PartOfSpeech=IN] [Text=bestiality CharacterOffsetBegin=140 CharacterOffsetEnd=150 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=151 CharacterOffsetEnd=153 PartOfSpeech=TO] [Text=begin CharacterOffsetBegin=154 CharacterOffsetEnd=159 PartOfSpeech=VB] [Text=with CharacterOffsetBegin=160 CharacterOffsetEnd=164 PartOfSpeech=IN] [Text=, CharacterOffsetBegin=164 CharacterOffsetEnd=165 PartOfSpeech=,] [Text=which CharacterOffsetBegin=166 CharacterOffsetEnd=171 PartOfSpeech=WDT] [Text=works CharacterOffsetBegin=172 CharacterOffsetEnd=177 PartOfSpeech=VBZ] [Text=for CharacterOffsetBegin=178 CharacterOffsetEnd=181 PartOfSpeech=IN] [Text=most CharacterOffsetBegin=182 CharacterOffsetEnd=186 PartOfSpeech=JJS] [Text=people CharacterOffsetBegin=187 CharacterOffsetEnd=193 PartOfSpeech=NNS] [Text=-LRB- CharacterOffsetBegin=193 CharacterOffsetEnd=194 PartOfSpeech=-LRB-] [Text=prudes CharacterOffsetBegin=194 CharacterOffsetEnd=200 PartOfSpeech=NNS] [Text=with CharacterOffsetBegin=201 CharacterOffsetEnd=205 PartOfSpeech=IN] [Text=weak CharacterOffsetBegin=206 CharacterOffsetEnd=210 PartOfSpeech=JJ] [Text=stomachs CharacterOffsetBegin=211 CharacterOffsetEnd=219 PartOfSpeech=NNS] [Text=-RRB- CharacterOffsetBegin=219 CharacterOffsetEnd=220 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=220 CharacterOffsetEnd=221 PartOfSpeech=.] 
root(ROOT-0, movie-3)
poss(movie-3, Its-1)
det(movie-3, a-2)
nsubj(tries-5, that-4)
rcmod(movie-3, tries-5)
aux(interesting-8, to-6)
cop(interesting-8, be-7)
xcomp(tries-5, interesting-8)
prep(interesting-8, by-9)
pobj(by-9, shocking-10)
dobj(shocking-10, you-11)
prep(shocking-10, with-12)
pobj(with-12, bestiality-13)
aux(begin-15, to-14)
xcomp(shocking-10, begin-15)
prep(begin-15, with-16)
punct(with-16, ,-17)
nsubj(works-19, which-18)
pcomp(with-16, works-19)
prep(works-19, for-20)
amod(people-22, most-21)
pobj(for-20, people-22)
punct(prudes-24, -LRB--23)
appos(people-22, prudes-24)
prep(prudes-24, with-25)
amod(stomachs-27, weak-26)
pobj(with-25, stomachs-27)
punct(prudes-24, -RRB--28)
punct(movie-3, .-29)

Sentence #3 (11 tokens):
But besides this there's not much to this movie.
[Text=But CharacterOffsetBegin=221 CharacterOffsetEnd=224 PartOfSpeech=CC] [Text=besides CharacterOffsetBegin=225 CharacterOffsetEnd=232 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=233 CharacterOffsetEnd=237 PartOfSpeech=DT] [Text=there CharacterOffsetBegin=238 CharacterOffsetEnd=243 PartOfSpeech=EX] [Text='s CharacterOffsetBegin=243 CharacterOffsetEnd=245 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=246 CharacterOffsetEnd=249 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=250 CharacterOffsetEnd=254 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=255 CharacterOffsetEnd=257 PartOfSpeech=TO] [Text=this CharacterOffsetBegin=258 CharacterOffsetEnd=262 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=263 CharacterOffsetEnd=268 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=268 CharacterOffsetEnd=269 PartOfSpeech=.] 
root(ROOT-0, But-1)
dep(But-1, besides-2)
pobj(besides-2, this-3)
expl('s-5, there-4)
rcmod(this-3, 's-5)
neg('s-5, not-6)
nsubj('s-5, much-7)
prep(much-7, to-8)
det(movie-10, this-9)
pobj(to-8, movie-10)
punct(But-1, .-11)

Sentence #4 (38 tokens):
Its not funny nor is it romantic so please somebody change that label , and since i already knew that bestiality is actually legal most places and a million dollar industry, i wasn't really shocked.
[Text=Its CharacterOffsetBegin=274 CharacterOffsetEnd=277 PartOfSpeech=PRP$] [Text=not CharacterOffsetBegin=278 CharacterOffsetEnd=281 PartOfSpeech=RB] [Text=funny CharacterOffsetBegin=282 CharacterOffsetEnd=287 PartOfSpeech=JJ] [Text=nor CharacterOffsetBegin=288 CharacterOffsetEnd=291 PartOfSpeech=CC] [Text=is CharacterOffsetBegin=292 CharacterOffsetEnd=294 PartOfSpeech=VBZ] [Text=it CharacterOffsetBegin=295 CharacterOffsetEnd=297 PartOfSpeech=PRP] [Text=romantic CharacterOffsetBegin=298 CharacterOffsetEnd=306 PartOfSpeech=JJ] [Text=so CharacterOffsetBegin=307 CharacterOffsetEnd=309 PartOfSpeech=RB] [Text=please CharacterOffsetBegin=310 CharacterOffsetEnd=316 PartOfSpeech=VB] [Text=somebody CharacterOffsetBegin=317 CharacterOffsetEnd=325 PartOfSpeech=NN] [Text=change CharacterOffsetBegin=326 CharacterOffsetEnd=332 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=333 CharacterOffsetEnd=337 PartOfSpeech=WDT] [Text=label CharacterOffsetBegin=338 CharacterOffsetEnd=343 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=344 CharacterOffsetEnd=345 PartOfSpeech=,] [Text=and CharacterOffsetBegin=346 CharacterOffsetEnd=349 PartOfSpeech=CC] [Text=since CharacterOffsetBegin=350 CharacterOffsetEnd=355 PartOfSpeech=IN] [Text=i CharacterOffsetBegin=356 CharacterOffsetEnd=357 PartOfSpeech=FW] [Text=already CharacterOffsetBegin=358 CharacterOffsetEnd=365 PartOfSpeech=RB] [Text=knew CharacterOffsetBegin=366 CharacterOffsetEnd=370 PartOfSpeech=VBD] [Text=that CharacterOffsetBegin=371 CharacterOffsetEnd=375 PartOfSpeech=IN] [Text=bestiality CharacterOffsetBegin=376 CharacterOffsetEnd=386 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=387 CharacterOffsetEnd=389 PartOfSpeech=VBZ] [Text=actually CharacterOffsetBegin=390 CharacterOffsetEnd=398 PartOfSpeech=RB] [Text=legal CharacterOffsetBegin=399 CharacterOffsetEnd=404 PartOfSpeech=JJ] [Text=most CharacterOffsetBegin=405 CharacterOffsetEnd=409 PartOfSpeech=RBS] [Text=places CharacterOffsetBegin=410 CharacterOffsetEnd=416 PartOfSpeech=NNS] [Text=and CharacterOffsetBegin=417 CharacterOffsetEnd=420 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=421 CharacterOffsetEnd=422 PartOfSpeech=DT] [Text=million CharacterOffsetBegin=423 CharacterOffsetEnd=430 PartOfSpeech=CD] [Text=dollar CharacterOffsetBegin=431 CharacterOffsetEnd=437 PartOfSpeech=NN] [Text=industry CharacterOffsetBegin=438 CharacterOffsetEnd=446 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=446 CharacterOffsetEnd=447 PartOfSpeech=,] [Text=i CharacterOffsetBegin=448 CharacterOffsetEnd=449 PartOfSpeech=FW] [Text=was CharacterOffsetBegin=450 CharacterOffsetEnd=453 PartOfSpeech=VBD] [Text=n't CharacterOffsetBegin=453 CharacterOffsetEnd=456 PartOfSpeech=RB] [Text=really CharacterOffsetBegin=457 CharacterOffsetEnd=463 PartOfSpeech=RB] [Text=shocked CharacterOffsetBegin=464 CharacterOffsetEnd=471 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=471 CharacterOffsetEnd=472 PartOfSpeech=.] 
root(ROOT-0, funny-3)
poss(funny-3, Its-1)
neg(funny-3, not-2)
cc(funny-3, nor-4)
aux(please-9, is-5)
nsubj(please-9, it-6)
dep(please-9, romantic-7)
advmod(please-9, so-8)
conj(funny-3, please-9)
nn(change-11, somebody-10)
dobj(please-9, change-11)
det(label-13, that-12)
dep(change-11, label-13)
punct(label-13, ,-14)
cc(label-13, and-15)
mark(knew-19, since-16)
nsubj(knew-19, i-17)
advmod(knew-19, already-18)
conj(label-13, knew-19)
mark(legal-24, that-20)
nsubj(legal-24, bestiality-21)
cop(legal-24, is-22)
advmod(legal-24, actually-23)
ccomp(knew-19, legal-24)
amod(places-26, most-25)
dobj(legal-24, places-26)
cc(places-26, and-27)
det(industry-31, a-28)
num(industry-31, million-29)
nn(industry-31, dollar-30)
conj(places-26, industry-31)
punct(places-26, ,-32)
nsubjpass(shocked-37, i-33)
auxpass(shocked-37, was-34)
neg(shocked-37, n't-35)
advmod(shocked-37, really-36)
rcmod(places-26, shocked-37)
punct(funny-3, .-38)

Sentence #5 (90 tokens):
But most people do not know this and i thought that how the movie evolved after the secret came out was just disappointing because it could have been one of those shocking eye opener movies that would teach the ignorant about a fetish that is very much a reality for(yes i did my homework)hundreds of thousands of people.So if you like watching bad movies that you can daydream about how you could make them better while watching them than sure,have fun doing that.
[Text=But CharacterOffsetBegin=477 CharacterOffsetEnd=480 PartOfSpeech=CC] [Text=most CharacterOffsetBegin=481 CharacterOffsetEnd=485 PartOfSpeech=JJS] [Text=people CharacterOffsetBegin=486 CharacterOffsetEnd=492 PartOfSpeech=NNS] [Text=do CharacterOffsetBegin=493 CharacterOffsetEnd=495 PartOfSpeech=VBP] [Text=not CharacterOffsetBegin=496 CharacterOffsetEnd=499 PartOfSpeech=RB] [Text=know CharacterOffsetBegin=500 CharacterOffsetEnd=504 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=505 CharacterOffsetEnd=509 PartOfSpeech=DT] [Text=and CharacterOffsetBegin=510 CharacterOffsetEnd=513 PartOfSpeech=CC] [Text=i CharacterOffsetBegin=514 CharacterOffsetEnd=515 PartOfSpeech=FW] [Text=thought CharacterOffsetBegin=516 CharacterOffsetEnd=523 PartOfSpeech=VBN] [Text=that CharacterOffsetBegin=524 CharacterOffsetEnd=528 PartOfSpeech=IN] [Text=how CharacterOffsetBegin=529 CharacterOffsetEnd=532 PartOfSpeech=WRB] [Text=the CharacterOffsetBegin=533 CharacterOffsetEnd=536 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=537 CharacterOffsetEnd=542 PartOfSpeech=NN] [Text=evolved CharacterOffsetBegin=543 CharacterOffsetEnd=550 PartOfSpeech=VBD] [Text=after CharacterOffsetBegin=551 CharacterOffsetEnd=556 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=557 CharacterOffsetEnd=560 PartOfSpeech=DT] [Text=secret CharacterOffsetBegin=561 CharacterOffsetEnd=567 PartOfSpeech=NN] [Text=came CharacterOffsetBegin=568 CharacterOffsetEnd=572 PartOfSpeech=VBD] [Text=out CharacterOffsetBegin=573 CharacterOffsetEnd=576 PartOfSpeech=RP] [Text=was CharacterOffsetBegin=577 CharacterOffsetEnd=580 PartOfSpeech=VBD] [Text=just CharacterOffsetBegin=581 CharacterOffsetEnd=585 PartOfSpeech=RB] [Text=disappointing CharacterOffsetBegin=586 CharacterOffsetEnd=599 PartOfSpeech=JJ] [Text=because CharacterOffsetBegin=600 CharacterOffsetEnd=607 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=608 CharacterOffsetEnd=610 PartOfSpeech=PRP] [Text=could CharacterOffsetBegin=611 CharacterOffsetEnd=616 PartOfSpeech=MD] [Text=have CharacterOffsetBegin=617 CharacterOffsetEnd=621 PartOfSpeech=VB] [Text=been CharacterOffsetBegin=622 CharacterOffsetEnd=626 PartOfSpeech=VBN] [Text=one CharacterOffsetBegin=627 CharacterOffsetEnd=630 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=631 CharacterOffsetEnd=633 PartOfSpeech=IN] [Text=those CharacterOffsetBegin=634 CharacterOffsetEnd=639 PartOfSpeech=DT] [Text=shocking CharacterOffsetBegin=640 CharacterOffsetEnd=648 PartOfSpeech=JJ] [Text=eye CharacterOffsetBegin=649 CharacterOffsetEnd=652 PartOfSpeech=NN] [Text=opener CharacterOffsetBegin=653 CharacterOffsetEnd=659 PartOfSpeech=NN] [Text=movies CharacterOffsetBegin=660 CharacterOffsetEnd=666 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=667 CharacterOffsetEnd=671 PartOfSpeech=WDT] [Text=would CharacterOffsetBegin=672 CharacterOffsetEnd=677 PartOfSpeech=MD] [Text=teach CharacterOffsetBegin=678 CharacterOffsetEnd=683 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=684 CharacterOffsetEnd=687 PartOfSpeech=DT] [Text=ignorant CharacterOffsetBegin=688 CharacterOffsetEnd=696 PartOfSpeech=JJ] [Text=about CharacterOffsetBegin=697 CharacterOffsetEnd=702 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=703 CharacterOffsetEnd=704 PartOfSpeech=DT] [Text=fetish CharacterOffsetBegin=705 CharacterOffsetEnd=711 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=712 CharacterOffsetEnd=716 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=717 CharacterOffsetEnd=719 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=720 CharacterOffsetEnd=724 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=725 CharacterOffsetEnd=729 PartOfSpeech=RB] [Text=a CharacterOffsetBegin=730 CharacterOffsetEnd=731 PartOfSpeech=DT] [Text=reality CharacterOffsetBegin=732 CharacterOffsetEnd=739 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=740 CharacterOffsetEnd=743 PartOfSpeech=IN] [Text=-LRB- CharacterOffsetBegin=743 CharacterOffsetEnd=744 PartOfSpeech=-LRB-] [Text=yes CharacterOffsetBegin=744 CharacterOffsetEnd=747 PartOfSpeech=UH] [Text=i CharacterOffsetBegin=748 CharacterOffsetEnd=749 PartOfSpeech=FW] [Text=did CharacterOffsetBegin=750 CharacterOffsetEnd=753 PartOfSpeech=VBD] [Text=my CharacterOffsetBegin=754 CharacterOffsetEnd=756 PartOfSpeech=PRP$] [Text=homework CharacterOffsetBegin=757 CharacterOffsetEnd=765 PartOfSpeech=NN] [Text=-RRB- CharacterOffsetBegin=765 CharacterOffsetEnd=766 PartOfSpeech=-RRB-] [Text=hundreds CharacterOffsetBegin=766 CharacterOffsetEnd=774 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=775 CharacterOffsetEnd=777 PartOfSpeech=IN] [Text=thousands CharacterOffsetBegin=778 CharacterOffsetEnd=787 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=788 CharacterOffsetEnd=790 PartOfSpeech=IN] [Text=people.So CharacterOffsetBegin=791 CharacterOffsetEnd=800 PartOfSpeech=NN] [Text=if CharacterOffsetBegin=801 CharacterOffsetEnd=803 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=804 CharacterOffsetEnd=807 PartOfSpeech=PRP] [Text=like CharacterOffsetBegin=808 CharacterOffsetEnd=812 PartOfSpeech=IN] [Text=watching CharacterOffsetBegin=813 CharacterOffsetEnd=821 PartOfSpeech=VBG] [Text=bad CharacterOffsetBegin=822 CharacterOffsetEnd=825 PartOfSpeech=JJ] [Text=movies CharacterOffsetBegin=826 CharacterOffsetEnd=832 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=833 CharacterOffsetEnd=837 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=838 CharacterOffsetEnd=841 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=842 CharacterOffsetEnd=845 PartOfSpeech=MD] [Text=daydream CharacterOffsetBegin=846 CharacterOffsetEnd=854 PartOfSpeech=VB] [Text=about CharacterOffsetBegin=855 CharacterOffsetEnd=860 PartOfSpeech=IN] [Text=how CharacterOffsetBegin=861 CharacterOffsetEnd=864 PartOfSpeech=WRB] [Text=you CharacterOffsetBegin=865 CharacterOffsetEnd=868 PartOfSpeech=PRP] [Text=could CharacterOffsetBegin=869 CharacterOffsetEnd=874 PartOfSpeech=MD] [Text=make CharacterOffsetBegin=875 CharacterOffsetEnd=879 PartOfSpeech=VB] [Text=them CharacterOffsetBegin=880 CharacterOffsetEnd=884 PartOfSpeech=PRP] [Text=better CharacterOffsetBegin=885 CharacterOffsetEnd=891 PartOfSpeech=JJR] [Text=while CharacterOffsetBegin=892 CharacterOffsetEnd=897 PartOfSpeech=IN] [Text=watching CharacterOffsetBegin=898 CharacterOffsetEnd=906 PartOfSpeech=VBG] [Text=them CharacterOffsetBegin=907 CharacterOffsetEnd=911 PartOfSpeech=PRP] [Text=than CharacterOffsetBegin=912 CharacterOffsetEnd=916 PartOfSpeech=IN] [Text=sure CharacterOffsetBegin=917 CharacterOffsetEnd=921 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=921 CharacterOffsetEnd=922 PartOfSpeech=,] [Text=have CharacterOffsetBegin=922 CharacterOffsetEnd=926 PartOfSpeech=VBP] [Text=fun CharacterOffsetBegin=927 CharacterOffsetEnd=930 PartOfSpeech=NN] [Text=doing CharacterOffsetBegin=931 CharacterOffsetEnd=936 PartOfSpeech=VBG] [Text=that CharacterOffsetBegin=937 CharacterOffsetEnd=941 PartOfSpeech=IN] [Text=. CharacterOffsetBegin=941 CharacterOffsetEnd=942 PartOfSpeech=.] 
root(ROOT-0, know-6)
cc(know-6, But-1)
amod(people-3, most-2)
nsubj(know-6, people-3)
aux(know-6, do-4)
neg(know-6, not-5)
nsubj(thought-10, this-7)
cc(this-7, and-8)
conj(this-7, i-9)
ccomp(know-6, thought-10)
mark(disappointing-23, that-11)
advmod(evolved-15, how-12)
det(movie-14, the-13)
nsubj(evolved-15, movie-14)
csubj(disappointing-23, evolved-15)
mark(came-19, after-16)
det(secret-18, the-17)
nsubj(came-19, secret-18)
advcl(evolved-15, came-19)
prt(came-19, out-20)
cop(disappointing-23, was-21)
advmod(disappointing-23, just-22)
ccomp(thought-10, disappointing-23)
mark(one-29, because-24)
nsubj(one-29, it-25)
aux(one-29, could-26)
aux(one-29, have-27)
cop(one-29, been-28)
advcl(disappointing-23, one-29)
prep(one-29, of-30)
det(movies-35, those-31)
amod(movies-35, shocking-32)
nn(movies-35, eye-33)
nn(movies-35, opener-34)
pobj(of-30, movies-35)
nsubj(teach-38, that-36)
aux(teach-38, would-37)
rcmod(movies-35, teach-38)
det(ignorant-40, the-39)
dobj(teach-38, ignorant-40)
prep(ignorant-40, about-41)
det(fetish-43, a-42)
pobj(about-41, fetish-43)
nsubj(reality-49, that-44)
cop(reality-49, is-45)
advmod(much-47, very-46)
advmod(reality-49, much-47)
det(reality-49, a-48)
rcmod(fetish-43, reality-49)
prep(reality-49, for-50)
punct(i-53, -LRB--51)
nn(i-53, yes-52)
pobj(for-50, i-53)
dep(i-53, did-54)
poss(homework-56, my-55)
dep(did-54, homework-56)
punct(homework-56, -RRB--57)
dobj(did-54, hundreds-58)
prep(hundreds-58, of-59)
pobj(of-59, thousands-60)
prep(thousands-60, of-61)
pobj(of-61, people.So-62)
mark(have-86, if-63)
nsubj(have-86, you-64)
prep(you-64, like-65)
pcomp(like-65, watching-66)
amod(movies-68, bad-67)
dobj(watching-66, movies-68)
mark(daydream-72, that-69)
nsubj(daydream-72, you-70)
aux(daydream-72, can-71)
dep(movies-68, daydream-72)
prep(daydream-72, about-73)
advmod(make-77, how-74)
nsubj(make-77, you-75)
aux(make-77, could-76)
pcomp(about-73, make-77)
dobj(make-77, them-78)
advmod(make-77, better-79)
mark(watching-81, while-80)
advcl(make-77, watching-81)
dobj(watching-81, them-82)
prep(watching-81, than-83)
pobj(than-83, sure-84)
punct(you-64, ,-85)
advcl(reality-49, have-86)
nsubj(doing-88, fun-87)
ccomp(have-86, doing-88)
dobj(doing-88, that-89)
punct(know-6, .-90)

